Gemfile.lock

```diff
@@ -48,6 +48,10 @@ GEM
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
+    appraisal (2.2.0)
```

Comment:
It's pretty standard for rubygems to put all lockfiles in `.gitignore` -- happy to remove this file if y'all prefer.